internal/backends/winit/build.rs

```diff
@@ -7,7 +7,7 @@ fn main() {
     // Setup cfg aliases
     cfg_aliases! {
        enable_skia_renderer: { any(feature = "renderer-skia", feature = "renderer-skia-opengl")},
-       skia_backend_opengl: { feature = "renderer-skia-opengl" },
+       skia_backend_opengl: { any(feature = "renderer-skia-opengl", all(feature = "renderer-skia", not(any(target_os = "macos", target_family = "windows", target_arch = "wasm32")))) },
        skia_backend_metal: { all(target_os = "macos", not(feature = "renderer-skia-opengl")) },
        skia_backend_d3d: { all(target_family = "windows", not(feature = "renderer-skia-opengl")) },
```

Comment:
Don't you need the `renderer-skia` feature still?

Reply:
It's not required here because the code that checks this is inside `#[cfg(enable_skia_renderer)]`. Conceptually right now it's sufficient to just use for example `caro run --bin slint-viewer --features renderer-skia-opengl`. If you think it's cleaner, then we can make `--features renderer-skia-opengl` a no-op and say that it *requires* also to specify `renderer-skia` (so `--features renderer-skia,renderer-skia-opengl`. I find that a little cumbersome, but I can change it if you prefer.